stories/select/__snapshots__/select.stories.storyshot

```diff
@@ -1,352 +1,13 @@
-  
-
-  <div
-    class="fd-popover"
-  >
-    
-  
-    <div
-      class="fd-popover__control"
-    >
-      
-     
-      <div
-        class="fd-select"
-      >
-        
-         
-        <div
-          aria-controls="h2A6CEW2"
-          aria-expanded="false"
-          aria-haspopup="listbox"
-          class="fd-select__control"
-          onclick="onControlClick('h0B6A326');"
-          role="combobox"
-          tabindex="0"
-        >
-          
-             
-          <span
-            class="fd-select__text-content"
-          />
-          
-             
-          <span
-            class="fd-button fd-button--transparent fd-select__button"
-          >
-            
-                
-            <i
-              class="sap-icon--slim-arrow-down"
-            />
-             
-            
-          </span>
-          
-         
-        </div>
-        
-      
-      </div>
-      
-  
-    </div>
-    
-  
-    <div
-      aria-hidden="true"
-      class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown"
-      id="h0B6A326"
-    >
-      
-     
-      <ul
-        aria-labelledby="h45336F3"
-        class="fd-list fd-list--dropdown"
-        id="h2A6CEW2"
-        role="listbox"
-        tabindex="-1"
-      >
-        
-        
-        <li
-          aria-label="None selected"
-          aria-selected="true"
-          class="fd-list__item is-selected"
-          role="option"
-          tabindex="0"
-        >
-          
-           
-          <span
-            class="fd-list__title"
-          >
-            
-        
-          </span>
-        </li>
-        
-        
-        <li
-          class="fd-list__item"
-          role="option"
-          tabindex="-1"
-        >
-          
-           
-          <span
-            class="fd-list__title"
-          >
-            List item 1
-          </span>
-          
-        
-        </li>
-        
-        
-        <li
-          class="fd-list__item"
-          role="option"
-          tabindex="-1"
-        >
-          
-           
-          <span
-            class="fd-list__title"
-          >
-            List item 2
-          </span>
-          
-       
-        </li>
-        
-        
-        <li
-          class="fd-list__item"
-          role="option"
-          tabindex="-1"
-        >
-          
-           
-          <span
-            class="fd-list__title"
-          >
-            List item 3
-          </span>
-          
-        
-        </li>
-        
-        
-        <li
-          class="fd-list__item"
-          role="option"
-          tabindex="-1"
-        >
-          
-           
-          <span
-            class="fd-list__title"
-          >
-            List item 4
-          </span>
-          
-        
-        </li>
-        
-     
-      </ul>
-      
-  
-    </div>
-    
-
-  </div>
-  
-
-  <br />
-  
+exports[`Storyshots Components/Select As Form Item 1`] = `
```

Comment:
Weird it deleted these, but we'll work on it as part of enabling the other tests